tests/test_integration/test_subscription_resource.py

```diff
@@ -0,0 +1,59 @@
+from tests.test_integration.test_case import YolaServiceTestCase
+
+from tests.test_integration.helpers import create_partner, create_user
+
+
+class TestYolaSubscription(YolaServiceTestCase):
+    """Yola: Subscription resource"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestYolaSubscription, cls).setUpClass()
+        cls.partner = create_partner(cls.service, properties={
+            'website': 'example.com',
+            'available_subscription_types': ['wl_basic', 'wl_lite']
+        })
+
+        user = create_user(cls.service, partner_id=cls.partner['id'])
+        cls.subscription = cls.service.create_subscription(
+            'wl_basic', user['id'], {'foo': 'bar'})
+
+    def test_can_create_subscriptions(self):
+        self.assertEqual(self.subscription['status'], 'active')
+
+    def test_can_list_subscriptions(self):
+        subs = self.service.list_subscriptions()['results']
+        self.assertTrue(len(subs) > 0)
+
+    def test_can_get_subscription(self):
+        sub = self.service.get_subscription(self.subscription['id'])
+        self.assertEqual(sub['id'], self.subscription['id'])
+
+    # Not working: https://github.com/yola/yolapy/issues/21
+    # def test_can_change_subscription_type(self):
```

Comment:
This may be the closest thing I've seen to acceptable commented-out code... but I still don't like it.

Instead how about:
* Commit the test as functional
* Purge the test with a commit message linking the blocking issue
* When (whoever) resolves the issue, they can revert the purge